CRM/Event/Form/Participant.php

```diff
@@ -1566,16 +1564,12 @@ protected function processContribution(
     $now = date('YmdHis');
     $receiptDate = NULL;
 
-    if (!empty($form->_values['event']['is_email_confirm'])) {
-      $receiptDate = $now;
```

Comment:
won't this stuff up L1595 tho?

Reply:
@seamuslee001 it is never set - it is probably being done wrong as a result - but this PR didn't seek to fix that, only to remove code that does nothing! I do hope to consolidate the two places that build the ContributionParams - (with a view to switching to v3 & then v4 order params) but thought this https://github.com/civicrm/civicrm-core/pull/28047 was enough to process in one hit on that front so have that work locally only

Reply:
right so essentially $receiptDate has been null then which seems odd / silly (thinking this is probably copy and paste from the front end forms

Reply:
@seamuslee001 yeah - it was previously shared code.... we have been unsharing code because our way of sharing requires more rather than less code logic :-)